Registration/DRR Generation/Python/DRR.py

```diff
@@ -0,0 +1,161 @@
+# This script generates Digitally Reconstructed Radiograph (DRR) from a given 
+# 3D image to given 2D image plane(s)
+
+# ISSUES - 
+# 1- I don't know how to set the direction of the output image
+
+import itk # imports insight Toolkit
+import numpy as np
+import sys
+from Functions import ChangeImageDirection
```

Comment:
This is fine, but in general you should avoid `from A import B`. Reduces the maintainability of your code. This is especially true for large files.

Reply:
@Besler  What do you suggest instead? If I want to get some defs from a class per say in the current folder or elsewhere, how can I do so by avoiding "from"? 
Thanks

Reply:
You can do something like
```python
# Imports
import Functions

# After much code...
Functions. ChangeImageDirection(image)
```

It's all good for now though.

Reply:
I can do it on my functions that are in my own directory. However, if I want to import the class the Christine has written in her folder, it doesn't work. 
But i get your point 👍